paddle/fluid/operators/slice_op.cc

```diff
@@ -121,6 +121,13 @@ class SliceOp : public framework::OperatorWithKernel {
           start = std::max(start, 0);
           end = std::max(end, 0);
           end = std::min(end, dim_value);
+
+          PADDLE_ENFORCE_LE(start, dim_value,
+                            platform::errors::InvalidArgument(
+                                "start should less than or equal to the "
```

Comment:
English grammar:

"start should less than" -> "start should be less than"

Reply:
Thanks, done.